Frends.FixedWidthFlatFile.ConvertToXML/Frends.FixedWidthFlatFile.ConvertToXML/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Frends.FixedWidthFlatFile.ConvertToXML.Definitions;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Xml;
+
+#pragma warning disable 1591
+
+namespace Frends.FixedWidthFlatFile.ConvertToXML
+{
+    public static class FixedWidthFlatFile
+    {
+        /// <summary>
+        /// This is a task that converts given value to json string.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.FixedWidthFlatFile.ConvertToXML)
+        /// </summary>
+        /// <param name="data">What value to convert.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>object { string Data }</returns>
+        public static Result ConvertToXML([PropertyTab] Input data, CancellationToken cancellationToken)
+        {
+            if(data.FileContent == null || data.FileContent.Count <= 0) throw new ArgumentNullException("FileContent not given. Cannot be empty.");
+
+            string xml = WriteToXmlString(data.FileContent, cancellationToken);
+            return xml != null ? new Result (xml) : throw new Exception("XML parse failed.");
+        }
+
+        private static string WriteToXmlString(List<Dictionary<string, dynamic>> data, CancellationToken cancellationToken)
+        {
+            try
+            {
+                using (var ms = new MemoryStream())
+                {
+                    using (var writer = new XmlTextWriter(ms, new UTF8Encoding(false)) { Formatting = Formatting.Indented })
+                    {
+                        writer.WriteStartDocument(); // start doc
+                        writer.WriteStartElement("Root");
+                        writer.WriteStartElement("Rows");
+
+                        foreach (var row in data)
+                        {
+                            cancellationToken.ThrowIfCancellationRequested();
```

Comment:
This line can be removed, since you are already checking the cancellation request in the innermost iteration.